courses/machine_learning/deepdive2/structured/solutions/3a_bqml_baseline_babyweight.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<p>Shouldn't the comment start with "--" instead "#" in a BiqQuery cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Done</p>